man/fi_endpoint.3.md

```diff
@@ -472,6 +472,11 @@ The following option levels and option names and parameters are defined.
   exchange between peer endpoints using the fi_connect, fi_accept,
   and fi_reject operations.  This option is read only.
 
+  The available space in CM messages may differ between endpoints created from
+  different domains. The size returned reflects the properties of the domain
+  from which the endpoint was created, except in the case of passive endpoints,
+  in which the size reflects the maximum size across all domains.
```

Comment:
Perhaps worth clarifying as "across all domains within the fabric to which the passive endpoint belongs"?  I.e., that it's not necessarily the max across all fabrics as well.